<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="title">
      <string>Wybór portu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Opis:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>Producent:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>Nr seryjny:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Lokalizacja:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="vidLabel">
        <property name="text">
         <string>Identyfikator:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="pidLabel">
        <property name="text">
         <string>ID produktu:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="parametersBox">
     <property name="title">
      <string>Parametry portu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="flowControlLabel">
        <property name="text">
         <string>Kontrola przepływu:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Bity stopu:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Bity danych:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parzystość:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Prędkość:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="additionalOptionsGroupBox">
     <property name="title">
      <string>Opcje dodatkowe</string>
     </property>
     <widget class="QDoubleSpinBox" name="comDelay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>3000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>200.000000000000000</double>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="ButtonBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Save</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
